### PR TITLE
c3: add new Queues and Cron Triggers templates

### PR DESCRIPTION
**Pull request by elithrar**
This PR adds new templates for:

* Queues, including both producer and consumer example handlers
* Scheduled Workers (Cron Triggers)

It also:

* Renames the "Common Worker functions" to a more descriptive "Example router & proxy Worker" (which is what it is)
* Adds both `queues` and `scheduled` to the e2e test harness.

This makes it easier for developers to get started and surfaces more Developer Platform products/features when first starting out via C3.

image

### CODE REVIEW

File: packages/create-cloudflare/e2e-tests/workers.test.ts [hunk 1]
```diff
@@ -74,4 +74,12 @@ describe("E2E: Workers templates", () => {
 	test("chatgptPlugin", async () => {
 		await runCli("chatgptPlugin");
 	});
+
+	test("queues", async () => {
+		await runCli("hello-world");
```
Comment (admah) [inline packages/create-cloudflare/e2e-tests/workers.test.ts:79]:
```suggestion
		await runCli("queues");
```

File: packages/create-cloudflare/e2e-tests/workers.test.ts [hunk 2]
```diff
@@ -74,4 +74,12 @@ describe("E2E: Workers templates", () => {
...
+	});
+
+	test("scheduled", async () => {
+		await runCli("hello-world");
```
Comment (admah) [inline packages/create-cloudflare/e2e-tests/workers.test.ts:83]:
```suggestion
		await runCli("scheduled");
```

**Comment by elithrar** [comment]
Incorporated Adam's suggestions + added a changeset.

File: packages/create-cloudflare/templates/queues/ts/src/worker.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ * - Run `npm run dev` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your worker in action
+ * - Run `npm run deploy` to publish your worker
+ *
+ * Learn more at https://developers.cloudflare.com/workers/
+ */
+
+export interface Env {
+	// Example binding to a Queue. Learn more at https://developers.cloudflare.com/queues/javascript-apis/
+	// MY_QUEUE: Queue;
```
Comment (RamIdeas) [inline packages/create-cloudflare/templates/queues/ts/src/worker.ts:16]: should this be uncommented?

**Review by RamIdeas — APPROVED** [review]

File: packages/create-cloudflare/templates/queues/ts/src/worker.ts (hunk 3 above)
Comment (elithrar) [reply packages/create-cloudflare/templates/queues/ts/src/worker.ts]: Fixed.

**Pull request merged**